Resources/Maps/Shuttles/rosebudmki.yml

```diff
@@ -783,6 +783,16 @@ entities:
     - type: RadiationGridResistance
     - id: RosebudMKII
       type: BecomesStation
+  - uid: 598
+    components:
+    - type: MetaData
+    - type: Transform
+    - type: Map
+    - type: PhysicsMap
+    - type: GridTree
+    - type: MovedGrids
+    - type: Broadphase
+    - type: OccluderTree
```

Comment:
are you certain it was re-saved as a grid? because this looks like it got saved as a map

Reply:
Whoops, sorry about that. Been doing regular mapping, so I was in the habit of doing savemap.